source/assets/js/playground/editor-setup.ts

```diff
@@ -34,6 +34,24 @@ import {
 } from '@codemirror/view';
 
 import {playgroundHighlightStyle} from './theme.js';
+import {EditorView} from 'codemirror';
+
+const syntax = new Compartment();
+
+const changeSyntax = (
```

Comment:
Please document this, and declare functions using `function` syntax.

I should probably bring in our common TS static check suite here...

Reply:
Updated, and I'm a fan of automating those checks!